src/simplewallet/simplewallet.cpp

```diff
@@ -362,6 +362,29 @@ bool simple_wallet::seed_set_language(const std::vector<std::string> &args/* = s
   return true;
 }
 
+bool simple_wallet::change_password(const std::vector<std::string> &args)
+{
+  // watch-only wallets don't need passwords, right?
+  if(m_wallet->watch_only())
+  {
+    fail_msg_writer() << tr("wallet is watch-only and has no seed");
+    return true;
+  }
+  
+  // prompts for a new password, this is not a new wallet so pass in false.
+  const auto pwd_container = tools::wallet2::password_prompt(false);
+  
+  if(!pwd_container)
+  {
+    fail_msg_writer() << tr("no password given");
```

Comment:
A failure message is already logged in `password_prompt`, I don't think this adds any value.